### PR TITLE
New MRI scanners in DeviceType

### PR DESCRIPTION
**Pull request by UlrikeS91**
from issue #19 

It seems like more common terms are: 
- open and closed MRI scanner
- standard-bore and wide-bore MRI scanner 

Therefore, I added "open MRI scanner", "standard-bore MRI scanner" and "wide-bore MRI scanner". The synonyms for each should give the alternatives with and with bore and/or closed.

### CODE REVIEW

File: instances/latest/terminologies/deviceType/openMRIScanner.jsonld [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/deviceType/openMRIScanner",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/DeviceType",
+  "definition": "An 'open MRI scanner' is a low-field scanner which has a magnetic top and bottom, but is otherwise open, increasing patient's comfort and unobstructed view of the scanning area. The typical magnetic strength lies between 0.2T and 0.3T.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "open MRI scanner",
```
Comment (lzehl) [inline instances/latest/terminologies/deviceType/openMRIScanner.jsonld:11]: change to "open-bore MRI scanner" make current name a synonym (I prefer to keep consistency for multiple entries with the same superclass)

File: instances/latest/terminologies/deviceType/openMRIScanner.jsonld [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/deviceType/openMRIScanner",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/DeviceType",
+  "definition": "An 'open MRI scanner' is a low-field scanner which has a magnetic top and bottom, but is otherwise open, increasing patient's comfort and unobstructed view of the scanning area. The typical magnetic strength lies between 0.2T and 0.3T.",
```
Comment (lzehl) [inline instances/latest/terminologies/deviceType/openMRIScanner.jsonld:7]: delete field strength. this should be specified elsewhere

File: instances/latest/terminologies/deviceType/standardBoreMRIScanner.jsonld [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/deviceType/standardBoreMRIScanner",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/DeviceType",
+  "definition": "A 'standard-bore MRI scanner' is a closed high-field scanner which features a magnet surrounding the patient creating a capsule-like space where the patient lies on. The typical magnetic strength lies between 1.0T and 3.0T.",
```
Comment (lzehl) [inline instances/latest/terminologies/deviceType/standardBoreMRIScanner.jsonld:7]: delete field strength. this should be specified elsewhere

File: instances/latest/terminologies/deviceType/wideBoreMRIScanner.jsonld [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/deviceType/wideBoreMRIScanner",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/DeviceType",
+  "definition": "A 'wide-bore MRI scanner' is a closed high-field scanner which features a widened bore compared to the standard-bore MRI scanner. The typical magnetic strength lies between 1.0T and 3.0T.",
```
Comment (lzehl) [inline instances/latest/terminologies/deviceType/wideBoreMRIScanner.jsonld:7]: delete field strength. this should be specified elsewhere

**Review by lzehl — CHANGES_REQUESTED** [review]
please also add superclass "closed-bore MRI scanner"

File: instances/latest/terminologies/deviceType/openMRIScanner.jsonld (hunk 1 above)
Comment (UlrikeS91) [reply instances/latest/terminologies/deviceType/openMRIScanner.jsonld]: done

File: instances/latest/terminologies/deviceType/wideBoreMRIScanner.jsonld (hunk 4 above)
Comment (UlrikeS91) [reply instances/latest/terminologies/deviceType/wideBoreMRIScanner.jsonld]: done

**Review by lzehl — APPROVED** [review]

**Pull request merged**